src/lib/types.d.ts

```
@@ -1,6 +1,33 @@
+	MULTIPLE = 'multiple-select'
+}
+
+export type TQuestion = {
+	id: number;
+	question_text: string;
+	instructions: string;
+	question_type: question_type_enum;
+	options: TOptions[];
+	subjective_answer_limit: number;
+	is_mandatory: boolean;
+	media: {} | null;
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)